docs/hiveutil.md

```diff
@@ -0,0 +1,47 @@
+# hiveutil
+
+The `hiveutil` CLI offers several commands to help manage clusters with Hive.
+
+To build the `hiveutil` binary, run `make hiveutil`.
+
+### Create Cluster
+
+The `create-cluster` command generates a `ClusterDeployment` and submits it to the Hive cluster using your current kubeconfig.
+
+To view what `create-cluster` generates, *without* submitting it to the API server, add `-o yaml` to the above command. If you need to make any changes not supported by `create-cluster` options, the output can be saved, edited, and then submitted with `oc apply`. This is also a useful way to generate sample yaml.
+
+By default, `create-cluster` assumes the latest Hive master CI build and the latest OpenShift stable release. `--release-image` can be specified to control which OpenShift release image to use.
```

Comment:
There is no assumption about the latest Hive master CI build.

Reply:
done